test/test-versions.js

```diff
@@ -30,85 +30,101 @@ function isValidVersion(browserIdentifier, version) {
+          hasErrors = true;
+        }
+        if ('version_removed' in statement && 'version_added' in statement) {
+          if (
+            typeof statement.version_added !== 'string' &&
+            statement.version_added !== true
+          ) {
+            logger.error(chalk`{red.bold ${relPath}} {red -} {red.bold version_added: "${statement.version_added}"}{red  is }{red.bold NOT}{red  a valid version number for }{red.bold ${browser}}{red  when }{red.bold version_removed}{red  is present}\n    {red Valid }{red.bold ${browser}}{red  versions are: ${validBrowserVersionsTruthy}}`);
             hasErrors = true;
-          }
-          if (!isValidVersion(browser, statement.version_removed)) {
-            console.error(chalk`{red.bold   version_removed: "${statement.version_removed}"}{red  is }{red.bold NOT}{red  a valid version number for }{red.bold ${browser}}`);
-            console.error(chalk`{red   Valid }{red.bold ${browser}}{red  versions are: ${validBrowserVersionsString}}`);
+          } else if (
+            typeof statement.version_added === 'string' &&
+            typeof statement.version_removed === 'string' &&
+            compareVersions(statement.version_added, statement.version_removed) >= 0
+          ) {
+            logger.error(chalk`{red.bold ${relPath}} {red -} {red.bold version_removed: "${statement.version_removed}"}{red  must be greater than }{red.bold version_added: "${statement.version_added}"}`);
             hasErrors = true;
           }
-          if ('version_removed' in statement && 'version_added' in statement) {
-            if (
-              typeof statement.version_added !== 'string' &&
-              statement.version_added !== true
-            ) {
-              console.error(chalk`{red.bold   version_added: "${statement.version_added}"}{red  is }{red.bold NOT}{red  a valid version number for }{red.bold ${browser}}{red  when }{red.bold version_removed}{red  is present}`);
-              console.error(chalk`{red   Valid }{red.bold ${browser}}{red  versions are: ${validBrowserVersionsTruthy}}`);
-              hasErrors = true;
-            } else if (
-              typeof statement.version_added === 'string' &&
-              typeof statement.version_removed === 'string' &&
-              compareVersions(statement.version_added, statement.version_removed) >= 0
-            ) {
-              console.error(chalk`{red.bold   version_removed: "${statement.version_removed}"}{red  must be greater than }{red.bold version_added: "${statement.version_added}"}`);
-              hasErrors = true;
-            }
-          }
         }
       }
     }
   }
 
+  return hasErrors;
+}
+
+/**
+ * @param {string} filename
+ */
+function testVersions(filename) {
+  const relativePath = path.relative(path.resolve(__dirname, '..'), filename);
+  const category = relativePath.includes(path.sep) && relativePath.split(path.sep)[0];
```

Comment:
I think `category` is not used.

Reply:
No need to keep a variable that isn't used -- removing now!